src/OpenApi/Command/OpenApiCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\OpenApi\Command;
```

Comment:
Why in a `Command` namespace? How do we do if we want a Laravel command too?

Reply:
let's handle that when we implement the Laravel command, my guess is that the code will be pretty similar and we shouldn't have 2 classes doing the same thing anyways.